internal/commands/scan_test.go

```diff
@@ -1187,3 +1188,17 @@ func TestValidateContainerImageFormat(t *testing.T) {
 		})
 	}
 }
+
+func TestRunScaResolverFileCleanup(t *testing.T) {
+	sourceDir := "/mock/sourceDir"
+	scaResolver := "/bin/echo"
+	scaResolverParams := ""
+	projectName := "testProject"
+
+	err := runScaResolver(sourceDir, scaResolver, scaResolverParams, projectName)
+
+	assert.NilError(t, err)
+
+	_, err = os.Stat(scaResolverResultsFile)
+	assert.Assert(t, os.IsNotExist(err), "Expected temp file to be deleted, but it exists")
+}
```

Comment:
As a continuation of the previous comment
check the logs to see that file really deleted